.github/BOTMETA.yml

```diff
@@ -287,7 +294,7 @@ files:
     maintainers: $team_linode
   $modules/cloud/linode/linode_v4.py:
     authors: decentral1se
-    maintainers: InTheCloudDan displague rmcintosh
+    maintainers: $team_linode
```

Comment:
Please create a new entry above the `$modules/cloud/linode/linode.py:` entry for the directory and set the maintainers there.

Reply:
done! i'll create an issue with suggestion adding authors to teams. If anybody agree, we can remove an extra visual noise and reduce BOTMETA size